rust/Cargo.toml

```diff
@@ -2,6 +2,14 @@
 name = "selenium-manager"
 version = "1.0.0-M2"
 edition = "2021"
+authors = ["Boni Garcia <@boni_gg>"]
```

Comment:
Let's put something like https://github.com/SeleniumHQ/docker-selenium/blob/trunk/Base/Dockerfile#L2 or maybe we add the TLC distribution list.